src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/InputNoWhitespaceAfterMethodRefBad.java

```diff
@@ -0,0 +1,20 @@
+package com.puppycrawl.tools.checkstyle.checks.whitespace;
+import java.util.function.Supplier;
+public class InputNoWhitespaceAfterMethodRefBad
+{
+  public static class SomeClass {
+    public static class Nested<V> {
+      private Nested() {
+      }
+    }
+  }
+
+  public static class Nested2<V> {
+  }
+
+  public <V> void methodName(V value) {
+    Supplier<?> t = Nested2<V>:: new;//16:33 violation
```

Comment:
wercker need to relaunched, please make some minor change ...... please add space before comment "//" and after ";"